src/Plugin.php

```diff
@@ -271,6 +275,23 @@ public static function log_cookie_consent() {
     wp_die();
   }
 
+  /**
+   * Sets site admin email.
+   *
+   * Allows to override the site admin email address to block
+   * error notices/emails sent from development or staging
+   * environments.
+   *
+   * @param string $value
+   *   Admin email saved in wp_options table.
+   *
+   * @return string
+   *   Admin email.
+   */
+  public static function customAdminEmail($value) {
```

Comment:
@juanlopez4691 Would have been nice if this email related adjustment was added to the existing Mail class:

https://github.com/netzstrategen/wordpress-core-standards/blob/master/src/Mail.php